### ISSUE TITLE
[question] Change "-" to "a" in the SingleInputDateRangeField component

### ISSUE THREAD

**Issue by MarcioVb** [mui/mui-x#10516]
### The problem in depth 🔍

Hey guys! 

I started using the SingleInputDateRangeField component for DateRangePicker, however in Brazil it is very common to use the letter "a" to separate the start date from the end date. Is there a way to change the "-" to "a"? 

I believe it is something simple, but very important for standardizing my system.

Thanks a lote!

### Your environment 🌎

<details>
  <summary>`npx @mui/envinfo`</summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @mui/envinfo` goes here.
```
</details>

**Search keywords**: SingleInputDateRangeField separate
**Order ID**: 66488

**Comment by flaviendelangle** [comment]
Hi,

You can find the doc [here](https://mui.com/x/react-date-pickers/custom-field/#customize-the-separator-of-multi-input-fields)

**Comment by LukasTy** [comment]
@flaviendelangle Didn't you mean to mention that it is not yet supported on single input range fields and we have an issue for that: https://github.com/mui/mui-x/issues/8369? 🤔

**Comment by flaviendelangle** [comment]
:facepalm: I read too fast

Yes for the single input rangs fields we do not have a solution for now.
Il will be solved in v7 with our multi-HTML-input approach.

This issue is a duplicate of #8369, I'm closing it

**Comment by LukasTy** [comment]
Hello, the new solution for fields with multi-HTML-elements is still in development.
The linked duplicate issue is also still not solved, hence, sorry, but there is still no option to do what you are asking. 🙈

**Comment by flaviendelangle** [comment]
Tbh looking at it today, I don't understand why I said that the new approach would solve this problem.

AFAIK, we just need a new prop on range fields that contains the separator between dates and use it inside the `fieldValueManager`

**Comment by MarcioVb** [comment]
Hello!
I'm happy to know that the solution is simpler than it seemed.
My entire application has range fields with the delimiter "a", only the date range that has the "-".
Thank you and I look forward to hearing from you.

**Comment by flaviendelangle** [comment]
I'll handle this one once #10649 is merged to avoid big conflicts :+1:
